samples/boards/mec15xxevb_assy6853/power_management/sample.yaml

```diff
@@ -1,6 +1,6 @@
 sample:
   name: MEC150x board sample power management tests
 tests:
-  sample.board.mec15xxevb_assy6853:
+  power_management.mec15xxevb_assy6853:
```

Comment:
should be 
`sample.board.mec15xxevb_assy6853.pm`

Reply:
I got you. But it looks weird to have sample.board.sample.board twice